cmd/training-operator.v1/main.go

```diff
@@ -81,8 +84,7 @@ func main() {
 		" Note: If you set another scheduler name, the training-operator assumes it's the scheduler-plugins.")
 	flag.StringVar(&namespace, "namespace", os.Getenv(EnvKubeflowNamespace), "The namespace to monitor kubeflow jobs. If unset, it monitors all namespaces cluster-wide."+
 		"If set, it only monitors kubeflow jobs in the given namespace.")
-	flag.IntVar(&monitoringPort, "monitoring-port", 9443, "Endpoint port for displaying monitoring metrics. "+
-		"It can be set to \"0\" to disable the metrics serving.")
+	flag.IntVar(&webhookServerPort, "webhook-server-port", 9443, "Endpoint port for the webhook server.")
```

Comment:
This is a bug and breaking change. So, can you open another PR?

Reply:
Reverted the flag to the old one.

Reply:
Not sure that this is a bug, because the old `Port` field was actually the webhook server port and not the monitoring port. Anyways the behaviour should be the same again and needs to be addressed separately.

Reply:
> Not sure that this is a bug, because the old Port field was actually the webhook server port and not the monitoring port.

Correct. I meant that removing the existing flag is breaking changes. So we should work on the breaking change at separate PR.